AvaliadorDisciplinas.FRONTEND/src/pages/UserPage/UserPage.module.css

```
@@ -0,0 +1,170 @@
+    padding: 1.25rem;
+  }
+  
+  .container .content {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+  
+  .container .content .header {
+    width: 70%;
+    height: 5%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    background: var(--secondary);
+    border-top-left-radius: 1.3rem;
+    border-top-right-radius: 1.3rem;
+    padding: 2rem;
+    box-shadow: rgba(0, 0, 0, 0.15) 0px 4px 4px;
+    margin-top: 11%;
+  }
+  
+  
+    .content .header .userNav{
+        display: flex;
+        flex-direction: row;
+        gap: 7.8rem;
+        max-height: min-content;
+        border-radius: 1rem 1rem 0rem 0rem;
+        list-style: none;
+        background-color: var(--secondary);
+        
+    }
+    .content .header .userNav li{
+        color: #FAFAFA;
+        font-family: 'Roboto';
+        font-style: normal;
+        font-weight: 700;
+        font-size: 1.5rem;
+        line-height: 1rem;
+        opacity: 0.7;
+        
+    
+    }
+
+    .content .header .userNav li:hover{
+
+        cursor: pointer;
+        transform: scale(1.075);
+    }
+
+   
+    .container .content .informs{
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        /* min-height: 80%; */
+        height: 100%;
+        
+    }
+    
+    .container .content  .informs .buttons{
+        margin-top: 1rem;
+        
+    }
+    
+    .container .content  .inputs{
+        display: flex;
+        flex-direction: column;
+        border-radius: 100rem;
+        
+    }
+    
+    .container .content .inputs input{
+        display: flex;
+        flex-direction: column;
+        margin-top: 5rem;  
+        border-radius: .5rem;
+        margin-top: 1.5rem;
+        padding: 1.5rem 1.5rem;
+        width: 40rem;
+        height: 3.5rem;
+        background-color: #E5E5E5;
+        border: solid #E5E5E5;;
+        align-self: center;
+        
+    }
+    .container .content .inputs input[type="email"]::placeholder,
+    .container .content .inputs input[type="nickName"]::placeholder,
+    .container .content .inputs input[type="userName"]::placeholder{
+        font-family: 'Roboto';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 1.2rem;
+        display: flex;
+        align-items: left;
+        justify-content: left;
+        color: #9E9E9E;
+        padding: 0;
+    }
+    
+    .container .content .userPhot{
+        margin-top: 2rem;
+        
+    
+    }
+    .container .content .itens {
+        width: 70%;
+        height: 81.5%;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        gap: 1rem;
+        border: 1px solid rgba(0, 0, 0, 0.15);
+        box-shadow: rgba(0, 0, 0, 0.15) 0px 4px 4px;
+        }
+    
+   
+    .container .content  .userBox a:hover{
```

Essa estilização não funciona porque a tag da imagem é `<img>` e não `<a>`
```suggestion
    .container .navMenu .userBox img
```